src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,52 @@
 package core.basesyntax.impl;
 
+import core.basesyntax.Pair;
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_CAPACITY = 10;
+    private final Pair[] pairs;
```

Comment:
you can create generic array and return data without converting it to the desired type using (V)
```suggestion
    private final Pair<K, V>[] pairs;
```